mmv1/templates/terraform/examples/compute_mig_resize_request.tf.erb

```diff
@@ -0,0 +1,69 @@
+resource "google_compute_region_instance_template" "a3_dws" {
+  name                 = "a3-dws"
+  region               = "us-central1"
+  description          = "This template is used to create a mig instance that is compatible with DWS resize requests."
+  instance_description = "A3 GPU"
+  machine_type         = "a3-highgpu-8g"
+  can_ip_forward       = false
+
+  scheduling {
+    automatic_restart   = false
+    on_host_maintenance = "TERMINATE"
+  }
+
+  disk {
+    source_image = "cos-cloud/cos-105-lts"
+    auto_delete  = true
+    boot         = true
+    disk_type    = "pd-ssd"
+    disk_size_gb = "960"
+    mode         = "READ_WRITE"
+  }
+
+  guest_accelerator {
+    type  = "nvidia-h100-80gb"
+    count = 8
+  }
+
+  reservation_affinity {
+    type = "NO_RESERVATION"
+  }
+
+  shielded_instance_config {
+    enable_vtpm                 = true
+    enable_integrity_monitoring = true
+  }
+
+  network_interface {
+    network = "default"
+  }
+}
+
+resource "google_compute_instance_group_manager" "a3_dws" {
+  name               = "a3-dws"
+  base_instance_name = "a3-dws"
+  zone               = "us-central1-a"
+
+  version {
+    instance_template = google_compute_region_instance_template.a3_dws.self_link
+  }
+
+  instance_lifecycle_policy {
+    default_action_on_failure = "DO_NOTHING"
+  }
+
+  wait_for_instances = false
+
+}
+
+resource "google_compute_resize_request" "<%= ctx[:primary_resource_id] %>" {
+  name                   = "<%= ctx[:vars]['resize_request_name'] %>"
```

Comment:
Thanks for the initial review @askubis!

Looks like the test isn't passing due to name being empty. In the yaml their isn't an existing variable for `resize_request_name` which is the reason for the error below.

```
Error: Error creating ResizeRequest: googleapi: Error 400: Invalid value for field 'resource.name': ''. Must be a match of regex '(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)', invalid
        
          with google_compute_resize_request.a3_resize_request,
          on terraform_plugin_test.tf line 60, in resource "google_compute_resize_request" "a3_resize_request":
          60: resource "google_compute_resize_request" "a3_resize_request" {
```

Other failing test isn't clear as this PR doesn't touch it and may just be a flaky test.

Reply:
@BBBmau Resolved the resize request name issue. The other test is not owned by our resource so I am not sure what that is.